### ISSUE TITLE
[Q]TSEVO on Multivariate Time Series

### ISSUE THREAD

**Issue by HaoWenJohn** [fzi-forschungszentrum-informatik/TSInterpret#10]
hello, thx for making this framework.
I'm having some difficulty when applying TSEVO on multivariate time series.
Everything works fine when used on a single channel dataset, but on multi-channel, the console always prints "Items are identical".
I find that TSEvo.explain returns a 1-d series as counterfactual
`return np.array(ep)[0][0], output,` 
and when call CF.plot_in_one,
`res = (item != exp).any(-1),
ind = np.where(res[0])`,
res becomes a 1-d array and makes len(ind)===0
which leads above problem.
I'm not sure if there is a problem with my understanding of the algorithm (it does return 'Items are identical') or if there is an identical program, so I hope you can help me.
sorry for my poor English /(ㄒoㄒ)/~~

**Comment by HaoWenJohn** [comment]
the same multi-channel data works on TSR and COMTE,I think my model construction and data processing should be no problem.

**Comment by JHoelli** [comment]
Hi , 

thanks for bringing this to our attention. You are compeltly right there was a minor bug in both the plot function and the return of TSEvo. I just fixed it and made a new release of TSInterpret (v.0.1.1 ), where this should be fixed. Also I added a <a href='https://github.com/fzi-forschungszentrum-informatik/TSInterpret/blob/v.0.1.1/docs/Notebooks/TSEvo_torch_Multivariate.ipynb'> notebook </a> for TSEvo in Multivariate Cases. If you still have issues, let me know.
